src/Domain/Infra/DependencyInjection/Bundle/ContainerHelper.php

```diff
@@ -9,8 +9,8 @@
 use Doctrine\ORM\Events as DoctrineOrmEvents;
 use MsgPhp\Domain\DomainIdentityMapInterface;
 use MsgPhp\Domain\Factory\{ClassMappingObjectFactory, ConstructorResolvingObjectFactory, EntityFactory, EntityFactoryInterface};
-use MsgPhp\Domain\Infra\Doctrine\DomainIdentityMap as DoctrineDomainIdentityMap;
-use MsgPhp\Domain\Infra\Doctrine\Mapping\{EntityFields, ObjectFieldMappingListener};
+use MsgPhp\Domain\Infra\Doctrine\{DomainIdentityMap as DoctrineDomainIdentityMap, EntityFieldsMapping};
```

Comment:
This notation is a pain to review :(

Reply:
i think im going to use the vendor namespace in general

```
use MsgPhp\Domain\Infra\Doctrine;
```

and have `Doctrine\Whateve` from there. That should reduce the no. of uses/aliases here and there by default :)